### PR TITLE
DOCS-126 review all ifdefs for TN. Fixed docs appropriately.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/tn_options.rst [hunk 1]
```diff
@@ -155,35 +165,19 @@ system.
...
-menubar, provides a shortcut to
-:menuselection:`System --> Support`.
-This screen can be used to verify the system license or to create a
-support ticket. Refer to :ref:`Support` for detailed usage
-instructions.
+Otherwise, set :guilabel:`Confirm` and click :guilabel:`SHUT DOWN` to
+halt the system. Shutting down the system will disconnect all clients,
+including the |web-ui|, and will power off the %brand% system. If the
+system has High Availability (HA) with :ref:`Failover` enabled, the
+system failsover to the passive |Ctrlr-term|.
```
Comment (Mrt134) [inline userguide/tn_options.rst]:
```suggestion
system failsover to the |ctrlr-term-standby|.
```
Replacement uses the management approved term for the "Standby" controller.

File: userguide/tn_options.rst [hunk 2]
```diff
@@ -55,97 +48,114 @@ the link has been clicked and Shell is open.
...
+reboot the system.
 Rebooting the system disconnects all clients, including the web
-administration GUI. The URL in the web browser changes, adding
-:literal:`/system/reboot/` to the end of the IP address. Wait a few
-minutes for the system to boot, then use the browser's Back button to
-return to the %brand% system's IP address and display the GUI login
-screen. If the login screen does not appear, access the system using
-IPMI to determine if a problem is preventing the system from resuming
-normal operation.
+administration GUI. Wait a few minutes for the system to boot. If the
```
Comment (Mrt134) [inline userguide/tn_options.rst]:
```suggestion
Wait a few minutes for the system to boot. If the
```

File: userguide/tn_options.rst [hunk 3]
```diff
@@ -55,97 +48,114 @@ the link has been clicked and Shell is open.
...
 periodically run :command:`zpool status` from `Shell`_
 until it is verified that the scrub or resilver process is complete.
-Once complete, the reboot request can be re-issued.
+Once complete, the restart request can be re-issued.
 
 Click the :guilabel:`Cancel` button to cancel the reboot request.
-Otherwise, click the :guilabel:`Reboot` button to reboot the system.
+Otherwise, set :guilabel:`Confirm` and click :guilabel:`Reboot` to
+reboot the system.
 Rebooting the system disconnects all clients, including the web
```
Comment (Mrt134) [inline userguide/tn_options.rst]:
```suggestion
Rebooting the system disconnects all clients, including the |web-ui|.
```

File: userguide/tn_options.rst [hunk 4]
```diff
@@ -55,97 +48,114 @@ the link has been clicked and Shell is open.
...
 
-Most FreeBSD command line utilities are available in Shell.
+.. index:: Log Out, Restart, or Shut Down
+.. _Log Out, Restart, or Shut Down:
+
+
+Log Out, Restart, or Shut Down
+------------------------------
+
+The |ui-power| button is used to log out of the web interface or
```
Comment (Mrt134) [inline userguide/tn_options.rst]:
```suggestion
The |ui-power| button is used to log out of the |web-ui| or
```

File: userguide/install.rst [hunk 5]
```diff
@@ -807,10 +807,6 @@ configuration. The system can be recovered with a few steps:
    into an older version of the operating system is performed,
    a configuration created in a later version cannot be restored.
 
-#ifdef freenas
-#include snippets/upgradingazfspool.rst
```
Comment (Mrt134) [inline userguide/install.rst]: If this text is obsolete or no longer used, should we remove the file too?

Comment (a-stjohn) [reply userguide/install.rst]: Must have been something I was investigating, deleted, still needed it, but forgot to replace it. I have put it back in. It appears it is still needed.

**Pull request merged**